CHANGES.md

```diff
@@ -34,7 +34,8 @@ Fixed Issues:
 * [#1213](https://github.com/ckeditor/ckeditor-dev/issues/1213): Fixed: Multiple images uploaded using [Upload Image](https://ckeditor.com/cke4/addon/uploadimage) plugin are randomly duplicated or mangled.
 * [#532](https://github.com/ckeditor/ckeditor-dev/issues/532): Fixed: Removed an outdated user guide link from the [About](https://ckeditor.com/cke4/addon/about) dialog.
 * [#1221](https://github.com/ckeditor/ckeditor-dev/issues/1221): Fixed: Invalid CSS loaded by [Balloon Panel](https://ckeditor.com/cke4/addon/balloonpanel) plugin when [`config.skin`](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.config-cfg-skin) is loaded using a custom path.
-* [#522](https://github.com/ckeditor/ckeditor-dev/issues/522): Fixed: Widget selection is not removed when widget is inside table cell with [Table Selection](https://ckeditor.com/cke4/addon/tableselection) plugin enabled. Fixes also [#1027](https://github.com/ckeditor/ckeditor-dev/issues/1027).
+* [#522](https://github.com/ckeditor/ckeditor-dev/issues/522): Fixed: Widget selection is not removed when widget is inside table cell with [Table Selection](https://ckeditor.com/cke4/addon/tableselection) plugin enabled.
+* [#1027](https://github.com/ckeditor/ckeditor-dev/issues/1027): Fixed: Cannot add images to table with [Table Selection](https://ckeditor.com/cke4/addon/tableselection) plugin enabled if image or widget is already present and focused inside it.
```

Comment:
I kind of feel that this title is a bit too verbose. Can we further simplify this title?

> Fixed: Cannot add images to table with Table Selection plugin enabled if image or widget is already present and focused inside it.

> Fixed: Cannot add multiple images to the table with Table Selection plugin in certain situations.

I feel like this proposition is in turn very generic, but I think it's actually better for a person checking changelog. 🙂

WDYT?

Reply:
Yeah, it's very verbose. If we don't want to go into such details, `certain situations` works here well.

Reply:
Corrected it as suggested (amend to previous commit).